### PR TITLE
fix: wait for Solana transaction confirmation

### PR DESCRIPTION
**Pull request by gabrielKerekes**
## Description

We currently do not wait for Solana tx confirmation. This means that when the user clicks the "Send" button we don't actually check if it's been successfully confirmed. It can happen that the tx won't be finalized e.g. if network is congested and the default fee is too low. In that case we display a notification that the tx has been submitted but the user doesn't see any notification about the transaction failing because of low fee. Since the transaction hasn't been confirmed it also doesn't show up in tx history.

This PR attempts to fix the issue, although it's more of a patch than a fix. During tx submission we would now also wait for it to be confirmed. If confirmation would fail then an error notification should be shown to the user. There's two issues with this fix though:
1. `sendAndConfirmRawTransaction` as it is used is currently deprecated. The same function with a different signature ([here](https://github.com/solana-labs/solana-web3.js/blob/1e328b9ea938473950c7337f7486b048bc1a72a4/packages/library-legacy/src/utils/send-and-confirm-raw-transaction.ts#L23)) should be used instead but we currently don't have the proper data in `pushTransaction` to set the `confirmationStrategy` - we would need `lastValidBlockHeight` which is not a part of the transaction and thus can't be determined in `pushTransaction`. `lastValidBlockHeight` is a part of the Suite store though, I'm just not sure how to properly pass it to `pushTransaction`.
2. Since tx confirmation actually takes a couple of seconds, the `TransactionReviewModal` stays open until the tx is confirmed. And so after the user clicks the "Send" button, there is currently no indication in the UI that something is happening. That's why I added the `isSending` state to `TransactionReviewOutputList` and disabled the Send button if `isSending` is true. I'm not sure if this is at all acceptable. A better solution would be to use a loading spinner but I'm not sure what spinner to use and if this would be acceptable even then.

A proper solution might be to implement pending transaction for Solana as e.g. similarly to Cardano. I have no idea how this works in Suite though. For this, issue 1 mentioned above would probably need to be resolved i.e. `lastValidBlockHeight` would need to be stored somewhere along with the transaction and then we would probably need to subscribe to [signatureSubscribe](https://solana.com/docs/rpc/websocket/signaturesubscribe) somewhere in the pending transactions logic.

### CODE REVIEW

**Comment by AdamSchinzel** [comment]
I think this is really on @Hermez-cz from product perspective if we want to merge this as a work around. I know that tx on Solana is mostly confirmed in a matter of few seconds but having the user stucked on tx confirm modal is not good UX in my opinion because we don't do this on any other chain.

It would be of course better to handle it as we handle every tx in pending state (so to add it to the list of pending txs of the account). Also I don't like usage of the deprecated method.

So to sum it up your thinking is good. Would you be able to research and implement the `confirmationStrategy` (probably harder part) and pending txs handling in Suite (probably the easier part). Thanks

**Comment by AdamSchinzel** [comment]
Closing in favor of https://github.com/trezor/trezor-suite/pull/11515
